### PR TITLE
Added authorization components

### PR DESCRIPTION
**Pull request by KobbAsa**

### CODE REVIEW

File: src/components/Auth/Auth.svelte [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        console.log('Signup data:', event.detail);
+    }
+
+    function switchToSignup() {
+        showLoginForm = false;
+    }
+
+    function switchToLogin() {
+        showLoginForm = true;
+    }
```
Comment (radichenko) [inline src/components/Auth/Auth.svelte]: It is better to use arrow functions

File: src/components/Auth/LoginForm.svelte [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    import { createEventDispatcher } from 'svelte';
+    import { slide } from 'svelte/transition';
+
+    export let email = '';
+    export let password = '';
+    const dispatch = createEventDispatcher();
+
+    function handleSubmit() {
+        dispatch('login', { email, password });
+    }
```
Comment (radichenko) [inline src/components/Auth/LoginForm.svelte]: It is better to use arrow functions

File: src/components/Auth/SignUpForm.svelte [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    export let confirmPassword = '';
+    const dispatch = createEventDispatcher();
+
+    function handleSubmit() {
+        if (password !== confirmPassword) {
+            alert('Passwords do not match');
+            return;
+        }
+        dispatch('signup', { firstName, lastName, email, password });
+    }
```
Comment (radichenko) [inline src/components/Auth/SignUpForm.svelte]: It is better to use arrow functions

**Review by radichenko — CHANGES_REQUESTED** [review]

File: src/components/Auth/Auth.svelte [hunk 4]
```diff
@@ -0,0 +1,81 @@
+<script>
+    import LoginForm from './LoginForm.svelte';
+    import SignupForm from './SignUpForm.svelte';
+    let showLoginForm = true;
+
+    function handleLogin(event) {
```
Comment (thebladehit) [inline src/components/Auth/Auth.svelte]: use arrow functions

**Review by thebladehit — CHANGES_REQUESTED** [review]

File: src/components/Auth/Auth.svelte (hunk 1 above)
Comment (SharpDevOps10) [inline src/components/Auth/Auth.svelte]: use arrow functions

**Review by SharpDevOps10 — CHANGES_REQUESTED** [review]

**Review by SharpDevOps10 — APPROVED** [review]

**Review by thebladehit — APPROVED** [review]

**Pull request merged**